src/overlays/actors/ovl_Dm_Ravine/z_dm_ravine.h

```diff
@@ -5,9 +5,14 @@
 
 struct DmRavine;
 
+typedef void (*DmRavineActionFunc)(struct DmRavine*, GlobalContext*);
+
 typedef struct DmRavine {
     /* 0x000 */ Actor actor;
-    /* 0x144 */ char unk_144[0x8];
+    /* 0x144 */ DmRavineActionFunc actionFunc;
+    /* 0x148 */ u8 state;
+    /* 0x149 */ u8 loaded;
+    /* 0x14A */ u8 pad_14A[2];
```

Comment:
I meant to mention this in my first review, but I think I didn't - since this is such a small pad and it's at the end of the actor, in my experience, these are not usually necessary. I don't know the details, but the pad is usually handled automatically. I would try removing pad_14A and see if you still get OK. If it is required, it should be something like `UNK_TYPE1 unk14A[2];`